vulnerabilities/package_managers.py

```diff
@@ -533,7 +533,7 @@ def trim_url_path(url_path: str) -> Optional[str]:
 
     @staticmethod
     def escape_path(path: str) -> str:
-        """escape uppercase in module/version name"""
+        """escape uppercase in module/version name. For example: escape_path("github.com/FerretDB/FerretDB") returns "github.com/!ferret!d!b/!ferret!d!b" """
```

Comment:
Do you mind to wrap this whole docstring to a shorter length, like 80 characters?